src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3989,7 +3994,6 @@ private static bool IsNegativeConstantForArraySize(BoundExpression expression)
 
                     var arguments = analyzedArguments.Arguments.ToImmutable();
                     var refKinds = analyzedArguments.RefKinds.ToImmutableOrNull();
-                    var argsToParamsOpt = memberResolutionResult.Result.ArgsToParamsOpt;
```

Comment:
>var argsToParamsOpt = memberResolutionResult.Result.ArgsToParamsOpt; [](start = 20, length = 68)

It is not clear why the local was removed.it looks like the value was reused in two places.